src/cloud_provider/aws/databases/utilities.rs

```diff
@@ -74,3 +83,82 @@ pub fn delete_terraform_tfstate_secret(
         }
     }
 }
+
+pub fn get_version_semver(version: &str) -> Result<VersionsNumber, StringError> {
```

Comment:
1/ I'd export this function outside of the Redis struct. 
2/ I'd rename it to `get_semver` or `get_version_number`. `version: &str` already gives details on the expected input.

Reply:
done